src/CodeViewer/components/BlockCodeViewer/BlockCodeViewer.tsx

```diff
@@ -20,6 +20,7 @@ const BlockCodeViewer: React.FC<IBlockCodeViewerProps> = ({ className, language,
   const [maxLines, setMaxLines] = React.useState<number | null>(null);
   const observerRef = React.useRef(new ObservableSet());
   const slicedBlocks = useSlicedBlocks(value, maxLines === null ? null : Math.max(0, maxLines - 1));
+  const linesNo = String(slicedBlocks !== null && maxLines !== null ? slicedBlocks.length * maxLines : 0).length;
```

Comment:
This is not `linesNumber` but st like `lineNumberCharacterCount` or `lineNumberLength` or similar.

Naming is important, it's already non-trivial to understand what the different variables do.